### ISSUE TITLE
Update app bar in home page

### ISSUE THREAD

**Issue by Amitkumarkoli** [Amitkumarkoli/House_Harbor#4]
Update and write more info of user in home page.

**Comment by noah-white-8** [comment]
Is this still open? How would I go about contributing? I'm new to this.

**Comment by Amitkumarkoli** [comment]
Yes it is still open , check out the contribution.md file for guidelines on how to contribute. Then I will assign it to u.

Don't hesitate to ask if you have any further questions or need clarification on anything.
